Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1354,15 +1354,20 @@ private string GetCommandLineArguments(string outputPath, EncodingOptions encodi
                 segmentFormat = "mpegts";
             }
 
+            var maxMuxingQueueSize = encodingOptions.MaxMuxingQueueSize >= 128 && encodingOptions.MaxMuxingQueueSize <= int.MaxValue
```

Comment:
Why `encodingOptions.MaxMuxingQueueSize <= int.MaxValue`?

Reply:
`The value for max_muxing_queue_size was 9999999999999999 which is not within -2147483648.000000 - 2147483647.000000`

FFmpeg only accepts a reasonable int32 value for max_muxing_queue_size .

Reply:
But `MaxMuxingQueueSize` is an int32, so how would it hold a larger value?

Reply:
Well. This check is redundant.